cmd/nodecmd/create.go

```diff
@@ -166,9 +175,41 @@ func preCreateChecks() error {
 	if customGrafanaDashboardPath != "" && !utils.FileExists(utils.ExpandHome(customGrafanaDashboardPath)) {
 		return fmt.Errorf("custom grafana dashboard file does not exist")
 	}
+
+	if useAWS {
+		if stringToAWSVolumeType(volumeType) == "" {
+			return fmt.Errorf("invalid AWS volume type provided")
+		}
+		if volumeType != constants.AWSVolumeTypeGP3 && throughput != constants.DefaultAWSThroughput {
```

Comment:
If you are going to use default for sanity checks, I'd recommend replacing the `125` in the config with it